android/src/main/java/com/revenuecat/purchases/common/common.kt

```diff
@@ -276,15 +275,15 @@ fun configure(
 
 // region private functions
 
-private fun getMakePurchaseErrorFunction(onResult: OnResult): (PurchasesError, Boolean) -> Unit {
+private fun getPurchaseErrorFunction(onResult: OnResult): (PurchasesError, Boolean) -> Unit {
     return { error, userCancelled -> onResult.onError(error.map(mapOf("userCancelled" to userCancelled))) }
 }
 
-private fun getMakePurchaseSuccessFunction(onResult: OnResult): (Purchase, PurchaserInfo) -> Unit {
+private fun getPurchaseCompletedFunction(onResult: OnResult): (Purchase?, PurchaserInfo) -> Unit {
```

Comment:
@vegaro let me know if I'm missing something, but it looks like the only significant change for android is  `Purchase` -> `Purchase?` here, right?

Reply:
Yes, that's it :) In Java there's more changes, but in Kotlin that's it